Modules/_operator.c

```diff
@@ -886,6 +886,14 @@ _operator__compare_digest_impl(PyObject *module, PyObject *a, PyObject *b)
     return PyBool_FromLong(rc);
 }
 
+/* helper function to make typing.NewType.__call__ method faster */
+
+static PyObject *
+functools_idfunc(PyObject* self, PyObject* x) {
```

Comment:
Perhaps remove the `functools_` prefix now that it's in `operator` module?

Reply:
Nice catch. Thanks)